### PR TITLE
OSD-6594 [ Added test to verify validation-webhook service is running.]

### PR DESCRIPTION
**Pull request by aliceh**
In this commit, I added tests to verify that the validation-webhook daemon set and service are running.

### CODE REVIEW

File: pkg/e2e/verify/pod_webhook.go [hunk 1]
```diff
@@ -250,3 +269,35 @@ func impersonateDedicatedAdmin(h *helper.H, user string) *helper.H {
...
+		Expect(err).NotTo(HaveOccurred(), "failed to get daemonset %v\n", daemonSetName)
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	}
+}
+
+// waitTimeoutForDaemonSetInNamespace waits the given timeout duration for the specified ds.
+func waitTimeoutForDaemonSetInNamespace(h *helper.H, daemonSetName string, namespace string, timeout time.Duration) error {
```
Comment (whereswaldon) [inline pkg/e2e/verify/pod_webhook.go:285]: Could I convince you to relocate this to a new method on the Helper type? `func (h *Helper) WaitForDaemonSet()` (and do the same for the service polling function below)? They seem useful to many people writing tests, not just this specific use-case.

**Review by whereswaldon — CHANGES_REQUESTED** [review]
I think this implementation totally makes sense, but we could make life easier for future test writers by exposing some of your helper functions to broader use.

File: pkg/e2e/verify/pod_webhook.go (hunk 1 above)
Comment (aliceh) [reply pkg/e2e/verify/pod_webhook.go]: @whereswaldon thanks for the suggestion!  I updated the commit accordingly.

**Review by whereswaldon — APPROVED** [review]
Thanks for exposing those helpers!

/lgtm

**Pull request merged**